command hook: respond to privmsg

Argentus opened the pull request:
Fixed bug: when YAUIB receives a private message, it responds to itself
instead of the sender.

Signed-off-by: Argentus radobezak@gmail.com

JulienPalard commented:
Could you please put doubles quotes around the `$s_user`? I already disabled glob expansion by using `-f` but better be pedantic with this.

Commit:
command hook: respond to privmsg

Fixed bug: when YAUIB receives a private message, it responds to itself
instead of the sender.

Signed-off-by: Argentus <radobezak@gmail.com>

Argentus commented:
Sure, done.
